content/en/docs/tutorials/github.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Setup GitHub Actions"
+subtitle: "Setup GitHub Actions for Continuous Integration"
+weight: 5
+draft: false
+description: >
+    A tutorial on how to set up Score with GitHub Actions.
+---
+
+You can use any Score implementation CLI in your GitHub Action workflows, for example, when you make a code change, and you want the Humanitec Platform Orchestrator to deploy your code to a new environment.
+
+The following is a guide to setting up the Score implementation CLI in your GitHub Action workflows.
+
+For the Score GitHub Action, see [Setup Score](https://github.com/score-spec/setup-score)
+
+## Usage
+
+To use the Score GitHub Action, add the following step to your [workflow](https://docs.github.com/en/actions/using-workflows/about-workflows):
+
+```yaml
+steps:
+  - uses: score-spec/setup-score@v2
+    with:
+      file: score-humanitec
```

Comment:
is this intended? `file = score-humanitec` and `run: score-compose` ?

Reply:
This is a copy-paste mistake. The CLI tool should be the one that is being installed with `setup-score` script.